hudi-common/src/main/java/org/apache/hudi/common/bootstrap/index/hfile/HFileBootstrapIndex.java

```diff
@@ -144,7 +144,7 @@ public BootstrapIndex.IndexReader createReader() {
 
   @Override
   public BootstrapIndex.IndexWriter createWriter(String bootstrapBasePath) {
-    return (IndexWriter) ReflectionUtils.loadClass("org.apache.hudi.common.bootstrap.index.hfile.HBaseHFileBootstrapIndexWriter",
+    return (IndexWriter) ReflectionUtils.loadClass("org.apache.hudi.common.bootstrap.index.hfile.HFileBootstrapIndexWriter",
```

Comment:
Can we avoid to hard code the class name, `HFileBootstrapIndexWriter` is visible to this class in compile time.